webapp/channels/src/components/advanced_text_editor/send_button/send_button.tsx

```diff
@@ -1,39 +1,35 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {FormEvent, memo} from 'react';
-import {useIntl} from 'react-intl';
+import classNames from 'classnames';
+import React, {memo, useCallback, useMemo} from 'react';
+import {defineMessage, useIntl} from 'react-intl';
+import {useSelector} from 'react-redux';
 import styled from 'styled-components';
 
 import {SendIcon} from '@mattermost/compass-icons/components';
+import type {SchedulingInfo} from '@mattermost/types/schedule_post';
+
+import {isSendOnCtrlEnter} from 'selectors/preferences';
+
+import {SendPostOptions} from 'components/advanced_text_editor/send_button/send_post_options';
+import WithTooltip from 'components/with_tooltip';
+import type {ShortcutDefinition} from 'components/with_tooltip/shortcut';
+import {ShortcutKeys} from 'components/with_tooltip/shortcut';
+
+import './style.scss';
```

Comment:
Could we avoid styling this with both Styled Components and CSS? I know we're not consistent with the two, but I think it'd be better to not mix the two on a single element

Reply:
Okay, IaI've removed the styled component now and we're all SCSS now 🙌